iam-policy-secretsmanager.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_iam_role_policy" "secretsmanager_for_ecs_readonly" {
+  name  = "secretsmanager"
```

Comment:
`secretsmanager ` --> `secretsmanager-ro`

Reply:
this is already qualified by role name? this is to just identify what kind of policy is that, why do we need this?

Reply:
okay. I thought it is going to name the policy.